docs/sources/v0.50.x/set-up/set-up-distributed-k6/troubleshooting.md

```diff
@@ -0,0 +1,251 @@
+---
+weight: 400
+title: Troubleshooting
+---
+
+# Troubleshooting
+
```

Comment:
The contents of this are a copy-paste from https://github.com/grafana/k6-operator/blob/main/docs/troublehooting.md

Reply:
Thanks for moving this over! 🙇